transloadit/client.py

```diff
@@ -190,7 +190,7 @@ def get_signed_smart_cdn_url(
             - template (str): Template slug or template ID
             - input (str): Input value that is provided as ${fields.input} in the template
             - url_params (Optional[dict]): Additional parameters for the URL query string. Values can be strings, numbers, booleans, arrays thereof, or None.
-            - expires_in (Optional[int]): Expiration time of signature in milliseconds. Defaults to 1 hour.
+            - expires_at_ms (Optional[int]): Timestamp in milliseconds since epoch when the signature is no longer valid. Defaults to 1 hour from now.
```

Comment:
```suggestion
            - expires_at_ms (Optional[int]): Timestamp in milliseconds since UNIX epoch when the signature is no longer valid. Defaults to 1 hour from now.
```

Trying to be more specific about which epoch